packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -81,17 +81,25 @@ export class Relayer {
       for (const relayableDeposit of relayableDeposits[l1Token]) {
         // If deposit quote time is before the bridgepool's deployment time, then skip it before attempting to calculate
         // the realized LP fee % as this will be impossible to query a contract for a timestamp before its deployment.
+        const latestBlockTime = Number((await this.l1Client.l1Web3.eth.getBlock("latest")).timestamp);
         if (
-          relayableDeposit.deposit.quoteTimestamp < this.deployTimestamps[relayableDeposit.deposit.l1Token].timestamp
+          relayableDeposit.deposit.quoteTimestamp < this.deployTimestamps[relayableDeposit.deposit.l1Token].timestamp ||
+          relayableDeposit.deposit.quoteTimestamp > latestBlockTime // + buffer?
         ) {
           this.logger.debug({
             at: "InsuredBridgeRelayer#Relayer",
-            message: "Deposit quote time < bridge pool deployment for L1 token, skipping",
+            message: "Deposit quote time < bridge pool deployment for L1 token or > latest block time, skipping",
             deposit: relayableDeposit.deposit,
             deploymentTime: this.deployTimestamps[relayableDeposit.deposit.l1Token].timestamp,
+            latestBlockTime,
           });
           continue;
         }
+        // Now, check if deposit.quoteTimestamp > latestBlockTime
```

Comment:
This PR is in draft mode until I implement this new edge case handling @chrismaree @mrice32 

Also similar situation in Disputer flow

Reply:
Done